src/app.tsx

```diff
@@ -1,8 +1,14 @@
 import * as React from 'react';
 import { hot } from 'react-hot-loader';
+import { MainComponent } from './pages/main';
+import { SideBarComponent } from './pages/sideBar';
```

Comment:
We're going to have a single page, so maybe a root `components` folder is enough.